cvat-canvas/README.md

```diff
@@ -37,6 +37,11 @@ Canvas itself handles:
         EXTREME_POINTS = 'By 4 points'
     }
 
+    enum CuboidDrawingMethod {
+        CLASSIC = 'Classic cuboid draw method',
+        CORNER_POINTS = 'By 4 points',
```

Comment:
What meaning do these messages mean? It isn't used in UI.

Reply:
@bsekachev, It's similar to `RectDrawingMethod` and have same naming and use cases. Can you explain problem with more details?

Reply:
@ActiveChooN 

No problem. Just question. 
I expect that for example line ``Classic cuboid draw method`` is used in UI somewhere (for example near the checkbox field). But it isn't. So, in this case it plays only role of comment. 

And now I see that for ``RectDrawingMethod`` we also do not use these value to display them in UI (but we might use), but values in UI are equal to values of the enumeration.